src/main/java/bio/terra/service/snapshotbuilder/utils/ConceptChildrenQueryBuilder.java

```diff
@@ -41,48 +47,50 @@ public class ConceptChildrenQueryBuilder {
    * c.ancestor_concept_id = conceptId AND c.descendant_concept_id != conceptId) GROUP BY
    * cc.concept_id, cc.concept_name ORDER BY cc.concept_name ASC
    */
-  public Query buildConceptChildrenQuery(SnapshotBuilderDomainOption domainOption, int conceptId) {
+  public Query buildConceptChildrenQuery(
+      SnapshotBuilderDomainOption domainOption, int parentConceptId) {
 
     // concept table and its fields concept_name and concept_id
-    TableVariable conceptTable =
+    TableVariable concept =
         TableVariable.forPrimary(TablePointer.fromTableName(CONCEPT, tableNameGenerator));
-    FieldVariable nameField = conceptTable.makeFieldVariable(CONCEPT_NAME);
-    FieldVariable idField = conceptTable.makeFieldVariable(CONCEPT_ID);
+    FieldVariable conceptName = concept.makeFieldVariable(CONCEPT_NAME);
+    FieldVariable conceptId = concept.makeFieldVariable(CONCEPT_ID);
 
     // concept_ancestor joined on concept.concept_id = ancestor_concept_id
-    TableVariable conceptAncestorTable =
+    TableVariable conceptAncestor =
         TableVariable.forJoined(
```

Comment:
This code joins through `concept_ancestor` and uses `concept_relationship` for its subquery. Maybe a comment would be helpful about why we use `ancestor` for rollup but `relationship` for finding the direct children?